lambda/shared/templates/helpers.ts

```diff
@@ -94,3 +94,11 @@ export const getIntegrationEmails = async (integration: Data, primaryEmailOnly =
 
   return [];
 };
+
+export const processIntegrationList = async (integrations: Array<Data>) => {
```

Comment:
wonder if the `one-liner` code can work as the same?
```
export const processIntegrationList =  (integrations: Array<Data>) => Promise.all(integrations.map(processRequest));
```

Reply:
I believe we use `Type[]` as a convention for array type as well.

Reply:
sure updated them all to maintain the consistency!